src/work/WorkScheduler.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "work/WorkScheduler.h"
+
+namespace stellar
+{
+WorkScheduler::WorkScheduler(Application& app)
+    : Work(app, "work-scheduler", BasicWork::RETRY_NEVER)
+{
+}
+
+WorkScheduler::~WorkScheduler()
+{
+}
+
+std::shared_ptr<WorkScheduler>
+WorkScheduler::create(Application& app, std::function<void()> wakeUpCallback)
```

Comment:
is/will be the wakeUpCallback parameter ever used?

Reply:
At this time, I don't think so. Going to remove it for now, and we can re-add if there's a use case.